babel-1.17: release-branch stay-behind changes (#824)
TranslatorSRI/Babel: docs/RunningBabel.md

       `screen -r`. You can also see a list of all running screens by running `screen -l`.

    5. Once the generation completes, all output files should be in the `babel_outputs` directory.

## Releasing a new Babel version

A full production run happens on an HPC system over many hours, and it almost always
surfaces problems that aren't visible from a local dry run: wrong memory settings,

> download endpoints that have moved or started blocking us, format changes upstream, and

Author: ```suggestion
download endpoints that have moved or require an API key, format changes upstream, and
```

### Which commits stay on the release branch

A commit can stay on the release branch if its entire effect fits in a single
release-note line, for example "updated the ENSEMBL dataset skip list", "bumped the
Biolink Model version", or "raised a rule's memory limit". These are the expected
running-a-build adjustments and reviewing them inline with the release is fine.

Everything else should move to its own branch off `main` and be reviewed as a normal
PR, so the change is documented, gets a real review, and earns its own release-note
entry. Related commits move together as one PR even if they were made days apart: all
the download-robustness work is one PR, all the DuckDB memory tuning is another, and so
on. Documentation and formatting commits travel with the code they describe rather than
staying behind on the release branch.

### Splitting the branch

1. **Classify by theme, not by date.** The commits interleave chronologically but group
   cleanly by the files they touch. List `git rev-list --no-merges main..<release-branch>`
   with each commit's changed files and bucket them (download robustness, a specific
   source's ingest, export/reporting, tooling, and so on).

> 2. **Make a backup ref** (`git branch <release-branch>-backup <release-branch>`) before

Author: ```suggestion
2. **Make a backup branch** (`git branch backups/<release-branch> <release-branch>`) before
```
